Metalama.Migration/src/Metalama.Migration/Aspects/Advices/OnEventRemoveHandlerAdvice.cs

```diff
@@ -9,7 +9,7 @@
 namespace PostSharp.Aspects.Advices
 {
     /// <summary>
-    /// In Metalama, implement the <see cref="IAspect{T}.BuildAspect"/> method and use <c>builder</c>.<see cref="IAspectBuilder.Advice"/>.<see cref="IAdviceFactory.OverrideAccessors(Metalama.Framework.Code.IEvent, string?, string?, string?, object?, object?)"/>
+    /// In Metalama, implement the <see cref="IAspect{T}.BuildAspect"/> method and use <c>builder</c>.<see cref="IAspectBuilder.Advice"/>.<see cref="IAdviceFactory.OverrideAccessors(Metalama.Framework.Code.IFieldOrPropertyOrIndexer,in Metalama.Framework.Advising.GetterTemplateSelector,string?,object?,object?)"/>
```

Comment:
This now references the OverrideAccessors method for wrong declaration kind - IFieldOrPropertyOrIndexer.